src/test/java/org/codehaus/mojo/license/utils/UrlRequesterTest.java

```diff
@@ -0,0 +1,46 @@
+package org.codehaus.mojo.license.utils;
+
+import java.io.File;
+import java.net.URL;
+
+import org.codehaus.mojo.license.utils.UrlRequester;
+import org.codehaus.plexus.util.FileUtils;
+import org.junit.Assert;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+
+public class UrlRequesterTest
+{
+
+    private static final String RESOURCE_NAME = "org/codehaus/mojo/license/utils/licenses.properties";
+
+    @Rule
+    public TemporaryFolder fileRule = new TemporaryFolder();
+
+    @Test
+    public void test_classpath_requester() throws Exception
```

Comment:
Plz. use camelCase as usual in Java method names.

Reply:
ok. Usually I prefer that way to write method test because increase readibility